src/IntegrationTests/ProjectionAdvanced.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AutoMapper.UnitTests;
+using Microsoft.EntityFrameworkCore;
+using Shouldly;
+using System.Linq;
+using Xunit;
+namespace AutoMapper.IntegrationTests;
+public class ProjectionAdvanced : IntegrationTest<ProjectionAdvanced.Initializer>
+{
+    protected override MapperConfiguration CreateConfiguration() => new(c => c.CreateProjection<Entity, Dto>().Advanced().ForAllMembers(o=>o.Ignore()));
```

Comment:
I'm not sure what extra members should we _really_ have for projections. But this is a decent workaround I believe. From #3929.